MANIFEST.in

```diff
@@ -2,6 +2,7 @@ include README.md
 include LICENCE
 include tox.ini
 include requirements.txt
+include requirements-dev.txt
```

Comment:
My personal preference is to just have `install_requires` and `tests_require` in `setup.py`